decentraland/unity-renderer
# fix: various fixes for emotes v2

## What does this PR change?

- Fixed confetti popper materials.
- Fixed emotes meshes not appearing in the backpack (they where being culled)

## How to test the changes?

- Enter: https://play.decentraland.org/?explorer-branch=fix/emotes-v2-more-fixes
- Play the emote in the backpack, the meshes should show
- Check `Detail Object Culling` in settings, toggle it (`false` to `true` or `true` to `false`)
- Press escape to exit the menu and then press F5
- Check that `Detail Object Culling` has the same value as before reloading
- Play the emote in the backpack, the meshes should show

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
